### PR TITLE
Darya Suhova

### PR DESCRIPTION
**Pull request by suhova**
Первые 10 задач из раздела Bioinformatics Stronghold. #ЯЧеловекПростой
image

### CODE REVIEW

File: darya_suhova/src/main/java/REVC.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class REVC {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        String result = new StringBuilder(input)
```
Comment (loki-1) [inline darya_suhova/src/main/java/REVC.java:8]: This solution requires construction of 5 additional lines, which can be suboptimal for large strings. Can it be done in different way?

**Review by loki-1 — CHANGES_REQUESTED** [review]

File: darya_suhova/src/main/java/CONS.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        'G', 2,
+        'T', 3
+    );
+    private static String[] dict = new String[]{"A", "C", "G", "T"};
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int cnt = 0;
+        int[][] result = new int[4][1000];
+        String input = scanner.nextLine();
```
[the remark this answers is not in the document]
Comment (suhova) [reply darya_suhova/src/main/java/CONS.java:19]: Yes. The input value in this problem always starts with an identifier to be skipped.

File: darya_suhova/src/main/java/FIB.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class FIB {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int months = scanner.nextInt();
+        int offsprings = scanner.nextInt();
+
+        long parent = 1;
```
[the remark this answers is not in the document]
Comment (suhova) [reply darya_suhova/src/main/java/FIB.java:9]: fixed. Maybe it looks clearer.

File: darya_suhova/src/main/java/REVC.java (hunk 1 above)
Comment (suhova) [reply darya_suhova/src/main/java/REVC.java]: fixed. Now I'm using arrays

File: darya_suhova/src/main/java/GC.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                if (max < percent) {
+                    max = percent;
+                    maxId = id;
+                }
+                id = input;
+                gcCount = 0;
+                length = 0;
+            } else {
+                length += input.length();
+                gcCount += input.length() - input
```
[the remark this answers is not in the document]
Comment (suhova) [reply darya_suhova/src/main/java/GC.java]: fixed. String copying removed

**Review by loki-1 — APPROVED** [review]

**Pull request merged**